macros/models/_project/balancer/balancer_protocol_revenue_macro.sql

```diff
@@ -0,0 +1,148 @@
+{% macro 
+    balancer_protocol_revenue_macro(
+        blockchain
+    ) 
+%}
+
+WITH pool_labels AS (
```

Comment:
I'm fine leaving these labels and prices CTEs on this query for now, but it's clearly a piece of code we repeat too much and could be abstracted. I'll add a task on Asana